model/job_queue/v1/queue_resource.model

```diff
@@ -27,10 +27,10 @@ resource Queue {
         out Kind String
         out HREF String
         in out Arguments String
-        out Attempts Integer
-        out AbandonedAt Date
+        in out Attempts Integer
```

Comment:
What does this mean? That the user pushing a job decides the maximum number of attempts to execute the job? Would be nice to have that explained in the documentation.

Reply:
AWS SQS permits to supply every field. None of them have special meaning. Both Attempts and AbandonedAt can be supplied at creation time.

Reply:
And technically we don't care what exactly client wants to specify

Reply:
These fields can be specified when you want to push the same job again to try more times. We're not advising for it, but we don't want to hide this as well.

Reply:
We don't want it to be in documentation. From end-user's point of view only `Arguments` are relevant at Push time

Reply:
I find it hard to understand this approach. I though we were building a service to manage jobs, and that SQS was just an implementation detail. But from your words it looks like the SQS details leak through the API of the service.

Anyhow, I assume you have considered this already, and decided it is a good way to proceed. I won't get in the way.